### PR TITLE
🛠 Fix the broken link for MVTec AD dataset

### LINKED ISSUE #250: Broken link for MVTec AD dataset
**Describe the bug**
- A clear and concise description of what the bug is.

For users first time download the MVTec AD dataset from `mvtec.py`, it shows `<urlopen error ftp error: error_perm('530 Login incorrect.')>`

**To Reproduce**
1. Make sure there is no MVTec under the dataset folder.
2. Run tools/train.py with MVTec AD dataset set up within the `config.yaml` for any available model.

**Expected behavior**
Start download MVTec AD dataset

**Screenshots**
image

**Hardware and Software Configuration**
- OS: Windows
- NVIDIA Driver Version [471.11]
- CUDA Version [e.g. 11.4]
- CUDNN Version [e.g. v11.4.120]

**Additional context**
- Add any other context about the problem here.

### PR DESCRIPTION
**Pull request by ABoLei**
# Description

Fix the broken link for MVTec AD dataset

- Fixes #250 

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the [pre-commit style and check guidelines](https://openvinotoolkit.github.io/anomalib/guides/using_pre_commit.html#pre-commit-hooks) of this project.
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

**Comment by samet-akcay** [comment]
@ABoLei CI fails due to long url line. Will you be able to run `black` formatter to fit to 120. Otherwise, I could do it.

**Comment by ABoLei** [comment]
@samet-akcay No problem, I will do it and make sure the codes are formatted correctly, thanks for remind :)

File: anomalib/data/mvtec.py [hunk 1]
```diff
@@ -380,7 +380,7 @@ def prepare_data(self) -> None:
             logging.info("Downloading the dataset.")
             with DownloadProgressBar(unit="B", unit_scale=True, miniters=1, desc="MVTec AD") as progress_bar:
                 urlretrieve(
-                    url=f"ftp://guest:GU.205dldo@ftp.softronics.ch/mvtec_anomaly_detection/{dataset_name}",
+                    url=f"https://www.mydrive.ch/shares/38536/3830184030e49fe74747669442f0f282/download/420938113-1629952094/{dataset_name}",
                     filename=self.root / dataset_name,
                     reporthook=progress_bar.update_to,
                 )
```
Comment (samet-akcay) [inline anomalib/data/mvtec.py:386]:
```
            logging.info("Downloading the dataset.")
            url = "https://www.mydrive.ch/shares/38536/3830184030e49fe74747669442f0f282/download/420938113-1629952094"
            dataset_name = "mvtec_anomaly_detection.tar.xz"
            with DownloadProgressBar(unit="B", unit_scale=True, miniters=1, desc="MVTec AD") as progress_bar:
                urlretrieve(
                    url=f"{url}/{dataset_name}",
                    filename=self.root / dataset_name,
                    reporthook=progress_bar.update_to,
                )
```

You could do something like this

Comment (ABoLei) [reply anomalib/data/mvtec.py:386]: Yes, I saw your  PR fixed the length of the url, graet job

Comment (samet-akcay) [reply anomalib/data/mvtec.py:386]: Should we close this one then?

Comment (ABoLei) [reply anomalib/data/mvtec.py:386]: sure, no problem.

**Pull request closed without merging**